.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -117,6 +117,13 @@ jobs:
         working-directory: lte/gateway
         run: |
           fab get_test_summaries:dst_path="test-results",sudo_tests=False,dev_vm_name="magma_deb"
+      - name: Publish Unit Test Results
```

Comment:
Is the step
```
      - name: Upload test results
        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # pin@v3
        if: always()
        with:
          name: test-results-${{ inputs.test_targets }}
path: lte/gateway/test-results/**/*.xml
```
still needed if we have this? Seems to me like we are uploading the same data twice, but this action also displays it nicely.

Reply:
I would leave it ...
1. other workflows also upload and publish (via the visual action) the results (consistent)
2. maybe people want to download the results in order to process them in other tooling